cw/__worker_main__.py

```diff
@@ -0,0 +1,29 @@
+from cw import callback
+from cw import logger
+from cw import listen
+from cw import open_connection
+from cw import close_connection
+from cw import RABBIT_HOST
+from cw import RABBIT_PORT
+from cw import RABBIT_QUEUE
+from cw import RABBIT_EXCHANGE
+from cw import RESULT_ROUTING_KEY
+import traceback
+import sys
+
+
+def main():
+    conn = None
+    try:
+        conn = open_connection(RABBIT_HOST, RABBIT_PORT)
+        listen(callback(conn, RABBIT_EXCHANGE, RESULT_ROUTING_KEY),
```

Comment:
the callback func doesn't use the connection, did you have another idea for it there?